dist/profile/manifests/mirrorbrain.pp

```diff
@@ -389,6 +389,30 @@
       },
     ],
     directories       => [
+      {
+        path            => "${docroot}/plugins",
+        options         => 'FollowSymLinks Indexes',
+        allow_override  => ['All'],
+        custom_fragment => '
+            MirrorBrainEngine On
+            MirrorBrainDebug Off
+            FormGET On
+            MirrorBrainHandleHEADRequestLocally Off
+
+            # we serve most files from mirrors, but as a fallback,
+            # this slow server has everything.
+            MirrorBrainFallback na us https://updates.azure.jenkins.io/
```

Comment:
I am not sure if I should redirect to http or https

Reply:
No. Jenkins cannot handle protocol upgrades. Pointless anyway, since the htaccess file only has http URLs.

Reply:
Switching to HTTP is probably best here